packages/kit/src/runtime/client/router.js

```diff
@@ -72,6 +72,20 @@ export class Router {
 		};
 	}
 
+	save_scroll_state() {
+		const new_state = {
+			...(history.state || {}),
+			'sveltekit:scroll': scroll_state()
+		};
+
+		// need this in case it was set in previous state
+		if (new_state['sveltekit:scroll'].x === 0 && new_state['sveltekit:scroll'].y === 0) {
+			delete new_state['sveltekit:scroll'];
+		}
```

Comment:
Not sure I understand the purpose of this?

Reply:
if an x and y coordinates are 0, that's the default position so we don't need to store it.